drivers/serial/uart_nrfx_uart.c

```diff
@@ -13,6 +13,8 @@
 #include <hal/nrf_gpio.h>
 
 
+static NRF_UART_Type *uart0_addr = (NRF_UART_Type *)CONFIG_UART_0_BASE;
```

Comment:
I'd add `const`. After the `*`, of course.

Reply:
done